src/SonarQube.Client/Api/V5_10/GetIssuesRequest.cs

```diff
@@ -51,6 +51,9 @@ public string Statuses
         [JsonIgnore] // We don't support the branch parameter in v5.10
         public string Branch { get; set; }
 
+        [JsonIgnore] // Not needed in old API
```

Comment:
It looks like `issues` was supported in v5.5 so the comment that it is not needed is a bit confusing. Do you mean we're choosing not to support it on versions before 7.2? (which I think is ok, I'm just unclear what the comment means).

Reply:
Ah, yes, bad wording.